### DISCUSSION TITLE
Can't reverify as a teacher due to browser location requirement

### DISCUSSION THREAD

**Discussion by e3-cerruti** [community/community#40282] [GitHub Education]
My school, for the safety of the students, does not allow computers to share their location with web browsers. As such I am unable to reverify my status as a teacher. Please advise.

**Comment by ossanna16** [comment]
Hi there @e3-cerruti and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by e3-cerruti** [comment]
I found a different device to verify throught.
